src/XMakeBuildEngine/Evaluation/ProjectParser.cs

```diff
@@ -186,6 +187,60 @@ private void ParseProjectElement(XmlElementWithLocation element)
             // so we have to set it now
             _project.SetProjectRootElementFromParser(element, _project);
 
+            if (element.HasAttribute(XMakeAttributes.sdk))
+            {
+
+                var sdksString = element.GetAttribute(XMakeAttributes.sdk);
+
+                var sdks = sdksString.Contains(";") ? sdksString.Split(';') : new[] {sdksString};
```

Comment:
>sdksString.Split(';') [](start = 54, length = 21)

You could also look into using ExpressionShredder.SplitSemiColonSeparatedList. Though it might be overkill for this situation.